docs/index.css

```diff
@@ -126,3 +126,7 @@ code {
     bottom: 3.45rem;
   }
 }
+
+.Toast:not(.Toast--dismissible) {
+  margin-bottom: 1rem;
```

Comment:
Not sure if I should add this to the Toast styles themselves or leave it up to the implementer to set a margin